swift3/bucket_db.py

```diff
@@ -220,6 +220,41 @@ def release(self, bucket):
         self.conn.delete(self._key(bucket))
 
 
+class BucketDbWrapper(object):
+    """
+    Memoizer for bucket DB. It is intended to have the same life cycle
+    as an S3 request.
+    """
+
+    def __init__(self, bucket_db):
+        self.bucket_db = bucket_db
+        self.cache = dict()
+
+    def get_owner(self, bucket, **kwargs):
+        cached = self.cache.get(bucket)
+        if cached:
+            return cached
+        owner = self.bucket_db.get_owner(bucket=bucket, **kwargs)
+        self.cache[bucket] = owner
+        return owner
+
+    def set_owner(self, bucket, owner, **kwargs):
+        res = self.bucket_db.set_owner(bucket=bucket, owner=owner)
```

Comment:
`**kwargs` is not used
```suggestion
        res = self.bucket_db.set_owner(bucket=bucket, owner=owner, **kwargs)
```